### PR TITLE
[CLOUDTRUST-5408] Support multiple back urls

### PR DESCRIPTION
**Pull request by basbeu**

### CODE REVIEW

File: api/account/api.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ type Configuration struct {
 	AutoIdentificationVoucherEnabled      *bool           `json:"auto_identification_voucher_enabled"`
 	AutoIdentificationAccountingEnabled   *bool           `json:"auto_identification_accounting_enabled"`
 	AutoIdentificationPrepaymentRequired  *bool           `json:"auto_identification_prepayment_required"`
-	AllowedBackURL                        *string         `json:"allowed_back_url"`
+	AllowedBackURLs                       []string        `json:"allowed_back_urls"`
```
Comment (fperot74) [inline api/account/api.go:103]: Do we want to support only new version? Selfservice will be ready in the same time?

File: api/management/api.go [hunk 2]
```diff
@@ -212,7 +212,7 @@ type RealmCustomConfiguration struct {
 	OnboardingClientID                  *string   `json:"onboarding_client_id"`
 	SelfRegisterGroupNames              *[]string `json:"self_register_group_names"`
 	BarcodeType                         *string   `json:"barcode_type"`
-	AllowedBackURL                      *string   `json:"allowed_back_url"`
+	AllowedBackURLs                     []string  `json:"allowed_back_urls"`
```
Comment (fperot74) [inline api/management/api.go:215]: Do we want to support only new version? Backoffice will be ready in the same time?

**Review by fperot74 — APPROVED** [review]

File: api/account/api.go (hunk 1 above)
Comment (basbeu) [reply api/account/api.go:103]: Selfservice and Backoffice have been updated

File: api/management/api.go (hunk 2 above)
Comment (basbeu) [reply api/management/api.go:215]: Selfservice and Backoffice have been updated

**Pull request merged**